src/test/java/guiTest/RegistrationTest.java

```diff
@@ -0,0 +1,109 @@
+package guiTest;
+
+import org.junit.Before;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+import pages.Registration;
+import utils.FunctionalTest;
+
+import static org.junit.Assert.*;
+import static utils.Constants.*;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+
+public class RegistrationTest extends FunctionalTest {
+
+    Registration registration = new Registration(driver);
+
+    @Before
+    public void init() {
+        driver.get(SIGN_UP_URL);
+    }
+
+    @Test
+    public void test01_noData_SignUpButtonIsDisabled() {
+        assertFalse(registration.isSignUpBtnEnabled());
+    }
+
+    @Test
+    public void test02_loginButtonIsEnabled() {
+        assertTrue(registration.isLoginBtnEnabled());
+    }
+
```

Comment:
No need for a separate test

Reply:
The method is verify is the web element (Login button) is enabled or disabled within the page, as we have a redirection to another page.